code/game/machinery/frame.dm

```diff
@@ -16,13 +16,17 @@
 	construction_frame_wall += cancel
 	construction_frame_floor += cancel
 
+
+//////////////////////////////
+// Frame Type Datum - Describes the frame structures that can be created from a frame item.
+//////////////////////////////
 /datum/frame/frame_types
 	var/icon/icon_override		// Icon to set on frame object when building. If null icon is unchanged.
 	var/name					// Name assigned to the frame object.
 	var/frame_size = 5			// Sheets of metal required to build.
 	var/frame_class				// Determines construction method.  "machine", "computer", "alarm", or "display"
 	var/circuit					// Type path of the circuit board that comes built in with this frame. Null to require adding a circuit.
-	var/frame_style = "floor"	// "floor" or "wall"
+	var/frame_style = FRAME_STYLE_WALL	// "floor" or "wall"
```

Comment:
Was this change from floor to wall intentional?

Reply:
No... it was not.
Although it has some great advantages in saved floor space! Imagine how spacious the labs will feel with wall mounted washing machines, wall mounted autolathes, wall mounted mass drivers!